django_rq/views.py

```diff
@@ -529,3 +536,20 @@ def enqueue_job(request, queue_index, job_id):
         'queue': queue,
     }
     return render(request, 'django_rq/delete_job.html', context_data)
+
+
+@never_cache
+@staff_member_required
+def stop_job(request, queue_index, job_id):
+    """Stop started job"""
+    if request.method == 'POST':
```

Comment:
We can use `require_POST` view decorator so we don't need to check for the method. https://docs.djangoproject.com/en/4.2/topics/http/decorators/#django.views.decorators.http.require_POST

Reply:
Done.